rpc/internal/system_metrics_service.go

```diff
@@ -26,9 +26,20 @@ func AttachSystemMetricsService(env cedar.Environment, s *grpc.Server) {
 	RegisterCedarSystemMetricsServer(s, srv)
 }
 
-//
-func (s *systemMetricsService) CreateSystemMetricRecord(context.Context, *SystemMetrics) (*SystemMetricsResponse, error) {
-	return nil, nil
+// CreateSystemMetricRecord creates a new system metrics record in the database.
+func (s *systemMetricsService) CreateSystemMetricRecord(ctx context.Context, data *SystemMetrics) (*SystemMetricsResponse, error) {
+	conf := model.NewCedarConfig(s.env)
+	if err := conf.Find(); err != nil {
+		return nil, newRPCError(codes.Internal, errors.Wrap(err, "problem fetching cedar config"))
+	}
+	if conf.Bucket.SystemMetricsBucketType == "" {
```

Comment:
I know this is probably copy-pasted from test results, but is it actually a reasonable assumption for the default bucket type to be a local filesystem bucket?

Reply:
Yeah for testing

Reply:
Sure, but is it actually reasonable to mix in test code with production code? Shouldn't the tests/local cedar specify this in the cedar configuration?

Reply:
done